libs/jt.timepicker/index.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
```

Comment:
Although we agree committing this plugin is the better of 2 evils until v7, should we prune it right down and get rid of these non-production files that we do not require?

Reply:
I thought about that but with the other things in `/libs` including non-production files and our pending move away from bower it felt unnecessary.

Reply:
Good point, agreed 🍻